### PR TITLE
Add support for class name

### PR DESCRIPTION
**Pull request by NullableInt**

### CODE REVIEW

**Review by marinafrydenb — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -25,9 +28,17 @@ To render a Material Design Icon use as follows:
...
+```liquid
+{% icon name_of_icon_you_want two-tone %}
+{% icon another_icon_perhaps round %}
+{% icon the_outlined_one outlined %}
+```
+
 ## Contributing
 
-Bug reports and pull requests are welcome on GitHub at https://github.com/[USERNAME]/jekyll-material-icon-tag. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [code of conduct](https://github.com/[USERNAME]/jekyll-material-icon-tag/blob/master/CODE_OF_CONDUCT.md).
+Bug reports and pull requests are welcome on GitHub at <https://github.com/[USERNAME>]/jekyll-material-icon-tag. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [code of conduct](https://github.com/[USERNAME]/jekyll-material-icon-tag/blob/master/CODE_OF_CONDUCT.md).
```
Comment (asbjornu) [inline README.md]:
```suggestion
Bug reports and pull requests are welcome on GitHub at <https://github.com/SwedbankPay/jekyll-material-icon-tag. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [code of conduct](https://github.com/SwedbankPay/jekyll-material-icon-tag/blob/master/CODE_OF_CONDUCT.md).
```

File: lib/jekyll-material-icon-tag.rb [hunk 2]
```diff
@@ -8,7 +8,20 @@ def initialize(tag_name, text, tokens)
   end
 
   def render(_)
-    "<i class=\"material-icons\">#{@text.strip}</i>"
+    iconname, type = parse_parameters @text
+
+    puts type
```
Comment (asbjornu) [inline lib/jekyll-material-icon-tag.rb]:
```suggestion
```

**Review by asbjornu — APPROVED** [review]

File: test/jekyll-material-icon-tag/index.md [hunk 3]
```diff
@@ -5,3 +5,5 @@ layout: home
 # About
 
 This page test the {% icon check %} gem.
+It also supports different {% icon alarm_on two-tone %} styles
+{% icon aspect_ratio round %}.
```
Comment (asbjornu) [inline test/jekyll-material-icon-tag/index.md:9]: Where or how do we see that this works?

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: test/jekyll-material-icon-tag/index.md (hunk 3 above)
Comment (NullableInt) [reply test/jekyll-material-icon-tag/index.md:9]: There are tests that checks that it is outputting correctly.

Comment (asbjornu) [reply test/jekyll-material-icon-tag/index.md:9]: Yes, I would just like to see the change as well.

**Review by asbjornu — APPROVED** [review]

**Pull request merged**